tools/osbuilder/rootfs-builder/nvidia/nvidia_chroot.sh

```diff
@@ -0,0 +1,339 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2024 NVIDIA Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+#!/bin/bash
+set -xe
+
+shopt -s nullglob
+shopt -s extglob
+
+run_file_name=$2
+run_fm_file_name=$3
+arch_target=$4
+driver_version="$5"
+driver_type="open"
+supported_gpu_devids="/supported-gpu.devids"
+
+APT_INSTALL="apt -o Dpkg::Options::='--force-confdef' -o Dpkg::Options::='--force-confold' -yqq --no-install-recommends install"
+
+export DEBIAN_FRONTEND=noninteractive
+
+install_nvidia_ctk() {
+	echo "chroot: Installing NVIDIA GPU container runtime"
+	apt list nvidia-container-toolkit-base -a
+	# Base  gives a nvidia-ctk and the nvidia-container-runtime
+	eval "${APT_INSTALL}" nvidia-container-toolkit-base
+}
+
+install_nvidia_fabricmanager() {
+	# if run_fm_file_name exists run it
+	if [ -f /"${run_fm_file_name}" ]; then
+		install_nvidia_fabricmanager_from_run_file
+	else
+		install_nvidia_fabricmanager_from_distribution
+	fi
+}
+
+install_nvidia_fabricmanager_from_run_file() {
+	echo "chroot: Install NVIDIA fabricmanager from run file"
+	pushd / >> /dev/null
```

Comment:
nit: I wouldn't pipe to /dev/null so we're consistent with other scripts in the repo.